.github/actions/templates/deployModule/action.yml

```diff
@@ -51,7 +51,10 @@ runs:
         # Load used funtions
```

Comment:
General question: Can you add an explaination of this replacement logic to the docs? E.g. in the `pipelineDesign` or `pipelineUsage` section. Probably a bit in both

Reply:
Yes will do.

Reply:
@MrMCake The current PR has gotten quite big with a lot of changes. I will do the update in a subsequent PR.

Reply:
addressed in the pipeline design and testing